website_sale_hide_price/views/website_sale_template.xml

```diff
@@ -38,10 +38,10 @@
             >pricelist,product,website,website.website_show_price,product.website_hide_price</attribute>
         </xpath>
         <xpath expr="//a[@id='add_to_cart']" position="before">
-            <t t-set="user_authenticated" t-value="user_id != website.user_id" />
```

Comment:
We don't need variable anymore since we are checking if the prices are hidden or not in another way (check _get_combination_info and _compute_website_show_price methods).